Android/Google.Android.DataTransport/source/TransportRuntime/Xamarin.Google.Android.DataTransport.TransportRuntime.csproj

```diff
@@ -38,8 +38,8 @@
     <Folder Include="Jars\" />
   </ItemGroup>
   <ItemGroup>
-    <LibraryProjectZip Include="..\..\externals\android\transport-runtime-2.2.0.aar">
-      <Link>Jars\transport-runtime-2.2.0.aar</Link>
+    <LibraryProjectZip Include="..\..\externals\android\transport-runtime-1.0.0.aar">
+      <Link>Jars\transport-runtime-1.0.0.aar</Link>
```

Comment:
Could you use $(PackageVersion) here,. in place of the `1.0.0`?

Reply:
Changed